src/app/components/publiclists/challengelist/challengelist.component.css

```diff
@@ -0,0 +1,3 @@
+section {
```

Comment:
Hey @Kajol-Kumari Why you need this `css` file? I think you should delete this file?

Reply:
@Sanji515 Deleted the file  as it was automatically generated by my editor(VSCode).